src/pages/publication/PublicationFormTabs.tsx

```diff
@@ -37,7 +38,13 @@ export const PublicationFormTabs: FC<PublicationFormTabsProps> = ({ handleTabCha
   } = values;
 
   return (
-    <Tabs variant="fullWidth" value={tabNumber} onChange={handleTabChange} aria-label="navigation" textColor="primary">
+    <Tabs
+      aria-label="navigation"
+      onChange={handleTabChange}
+      scrollButtons={isMobile ? 'on' : 'off'}
```

Comment:
vil ikke `auto` virke vel så fint? 

> Determine behavior of scroll buttons when tabs are set to scroll:
- `auto` will only present them when not all the items are visible

Reply:
kanskje enda bedre også, om den tar hensyn til bytte mellom portrait og landscape, og resizing

Reply:
smart! så ikke at det var en option!

Reply:
blir samme med auto. må refreshe for å få scroll-buttons